src/main.rs

```diff
@@ -187,7 +196,23 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             });
         };
 
+        {
+            let mut autoreload_tls_interval = tokio::time::interval(tokio::time::Duration::from_secs(60 * 60)); // update root certificates every hour
```

Comment:
I'm not sure we want this. Do you have very short-lived certificates in your setup? Typical certificates are issued for at least 3 months (e.g. let's encrypt) and private CAs can issue arbitrary length certs. In any case, this shouldn't be done automatically every second but with a command issued to the pooler instead, e.g. you can do this when we issue a `RELOAD` to the admin database.

Reply:
What makes you think the certificates will be updated every second? I made them update hourly.

Reply:
Sorry, that was a typo on my part. Every hour. But even that seems odd, that's not something that happens as far as I know, and even if some environments issue very short-lived certs, I think we should make this an explicit command for Pgcat to reload them, and not have to wait every hour. Some environments for example may choose to rotate certs more frequently, and the hourly frequency is not often enough for them. So my proposal is reload certs when someone issues a `RELOAD` command to the pooler or a SIGHUP, basically in `reload_config`.

Reply:
I agree with you. Good, then I will correct it and prepare a new version